STYLE.md

```diff
@@ -1,3 +1,13 @@
+## General file style
+
+- General style guidelines -- like newlines at the end of file -- are linted
+for using [pre-commit](https://pre-commit.com/)
+- The rules enfornced are available in the
```

Comment:
```suggestion
- The rules enforced are available in the
```

Reply:
hmmm spell checker from all_lint should catch that. Let me take a look at that.

Reply:
nvm we don't have that on. Will add that.